statki.html

```diff
@@ -9,11 +9,20 @@
      <meta http-equiv="X-Ua-Compatible" content="IE=edge,chrome=1">
      <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
      <link href="https://fonts.googleapis.com/css?family=Quicksand:500&display=swap" rel="stylesheet">
+     <link href="https://fonts.googleapis.com/css?family=Bungee|Bungee+Inline&display=swap" rel="stylesheet">
      <link rel="stylesheet" href="shipcss.css">
   </head>
   <body>
+        <div id="startpage">
+            <h1>Statki</h1>
+            <h3>Zapraszamy do najlepszej  wersji</br>przegądarkowej gry w statki!</br></br></br>Wybierz sposób gry:</h3>
+            <button id="onePlayer">Jeden Gracz</button>
+            <button id="twoPlayers">Dwóch Graczy</button>
+            <button id="aiPlayer">Zagraj z AI</button>
+            <img id="radarImg" src="../ships/photo/radar-pngrepo-com.png" alt="radar">

```

Comment:
Please change path to `"../photo/radar.png"`
And filename respectively.

Reply:
solved